### PR TITLE
`Development`: Update flake check to use build number

### PR DESCRIPTION
**Pull request by heddendorp**
Also adds the bamboo scripts to vcs

#### General

- [ ] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).

### Motivation and Context

The selected build for comparison, the last successful build, could previously end up being one later than the analyzed build.

### Description

This is a small change that adds the bamboo scripts into version control and forwards the current build number to the flake checker. This is to make sure the correct reference is used.

#### Code Review
- [ ] Review 1
- [ ] Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2
#### Exam Mode Test
- [ ] Test

### CODE REVIEW

File: src/main/docker/cypress/Cypress-Dockerfile [hunk 1]
```diff
@@ -1,5 +1,4 @@
 ARG CYPRESS_BROWSER=node18.6.0-chrome105-ff104
 FROM cypress/browsers:${CYPRESS_BROWSER}
-RUN export DEBIAN_FRONTEND=noninteractive && \
-    apt update && \
-        apt -y install openjdk-17-jdk
+RUN apt-get update
```
Comment (4ludwig4) [inline src/main/docker/cypress/Cypress-Dockerfile:3]: Why not run _apt-get update, install, clean and removing apt_ install files in one step like in the `Coverage-Dockerfile`?

**Review by TheZoker — DISMISSED** [review]
Code LGTM 👍

File: src/main/docker/cypress/Cypress-Dockerfile (hunk 1 above)
Comment (heddendorp) [reply src/main/docker/cypress/Cypress-Dockerfile:3]: Will do

**Review by 4ludwig4 — CHANGES_REQUESTED** [review]
I already resolved all the Codacy Issues which appear in this PR and also cleaned up the environments. So you can maybe streamline this. All other streamlining I'll do in my big docker compose PR after this gets merged.

**Pull request closed without merging**